app/Space.php

```diff
@@ -63,12 +63,12 @@ public function monthlyBalance($year, $month) {
         }
     }
 
-    public function monthlyRecurrings($year, $month) {
+    public function monthlyRecurrings($earning, $year, $month) {
```

Comment:
I assume that with the `$earning` boolean, you want to determine whether or not you're fetching earnings **or** spendings.

However the interpretation of this parameter, to me at least, is that you mean "should I **include** earnings or not". I would just make it a `$transactionType` parameter that can either be 0 or 1 (0 for spendings, 1 for earnings), or "earnings" or "spendings.